backend/config/corpora-api.yml

```diff
@@ -413,6 +413,65 @@ paths:
         "400":
           $ref: "#/components/responses/400"
 
+    post:
+      tags:
+        - collections
+      summary: Create a collection
```

Comment:
add the security scheme somewhere around here. Once you do and pass in a token, add the user field to `corpora.lambdas.api.v1.collection.create_collection`